cluster/manifests/skipper/configmap-open-policy-agent.yaml

```diff
@@ -0,0 +1,40 @@
+{{ if eq .ConfigItems.skipper_open_policy_agent_enabled "true" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    application: skipper-ingress
```

Comment:
Please add also
`component: ingress`
or can we offload this to `component: routesrv` (I guess not)?

Reply:
Yes, changed to `component: ingress`